rosidl_generator_cpp/resource/msg__struct.hpp.em

```diff
@@ -43,6 +43,7 @@ cpp_full_name_with_alloc = '%s<ContainerAllocator>' % (cpp_full_name)
 #include <memory>
 #include <string>
 #include <vector>
+#include <rosidl_generator_cpp/bounded_vector.hpp>
```

Comment:
It is only needed for messages which contain bounded arrays. But the same is true for the other includes above, e.g. `array`, `string`, `vector`.